x/pubkey/types/params.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	DefaultActivationBlockDelay       = 11520 // roughly 1 day
```

Comment:
Do we want to change this?

Reply:
For testing yes. For mainnet release we would have to go over the default parameters again. Added that to the v1.0.0 release checklist on Notion.